editor/run/game_view_plugin.cpp

```diff
@@ -1074,6 +1116,27 @@ GameView::GameView(Ref<GameViewDebugger> p_debugger, EmbeddedProcessBase *p_embe
 	next_frame_button->set_accessibility_name(TTRC("Next Frame"));
 	next_frame_button->set_shortcut(ED_GET_SHORTCUT("editor/next_frame_embedded_project"));
 
+	speed_state_button = memnew(MenuButton);
+	main_menu_hbox->add_child(speed_state_button);
+	speed_state_button->set_text(TTRC("1.00x"));
```

Comment:
```suggestion
	speed_state_button->set_text(TTRC(U"1.00×"));
```

Also, I'm not sure if this should be a translatable string as it doesn't appear to have anything to localize.

Reply:
I agree, no need to translate numbers.